src/note_encryption/orchard_domain.rs

```diff
@@ -0,0 +1,79 @@
+//! Represents the Orchard protocol domain specifics required for note encryption and decryption.
+//!
+//! Defines OrchardDomain triat with the constants and methods needed to handle the
+//! various sizes and types of note components. Actually this trait represents the difference
+//! between both Vanilla and ZSA variantions of the Orchard protocol.
```

Comment:
//! The OrchardDomain trait represents the difference between the Orchard and the OrchardZSA 
//! encryption and decryption procedures.

Reply:
Fixed.